.travis.yml

```diff
@@ -14,6 +14,15 @@ before_script:
 script:
   - gulp
 
+deploy:
+  provider: npm
+  email: "john@radify.io"
```

Comment:
I've just set up `shipping@radify.io` as an email group that we can use for stuff like this, so that no one becomes a single point of failure.